cost/azure/idle_compute_instances/azure_idle_compute_instances.pt

```diff
@@ -0,0 +1,324 @@
+name "Azure Idle Compute Instances"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report on any instances that have lower than specified average CPU utilization and delete them with approval"
```

Comment:
change short_description to: Checks for Azure Compute Instances that are idle for the last 30 days and terminates them after approval.